styles/base/inputs/_svg-icon-input.scss

```diff
@@ -12,24 +11,19 @@
   max-width: $sprk-input-container-narrow-max-width;
 }
 
-// TODO: .sprk-b-InputContainer__icon-container .sprk-c-Icon is a fallback to be deprecated later
-.sprk-b-InputContainer__icon-container .sprk-c-Icon,
-.sprk-b-TextInputIconContainer .sprk-c-Icon,
+.sprk-b-InputContainer__icon-container .sprk-b-InputContainer__icon,
+.sprk-b-TextInputIconContainer .sprk-b-InputContainer__icon,
 .sprk-b-InputContainer__icon {
   margin-left: $sprk-input-icon-offset-x;
-  // TODO: We should be using margin bottom, not margin top
   margin-top: $sprk-input-icon-offset-y;
   position: absolute;
   z-index: $sprk-input-icon-z-index;
 }
 
-// Specificity needed to override {.sprk-b-TextInputIconContainer .sprk-c-Icon}
-/// For a SVG icon inside of input that is positioned to the right.
 /// @name .sprk-b-InputContainer__icon--right
 .sprk-b-InputContainer__icon-container .sprk-b-InputContainer__icon--right {
-  top: $sprk-input-has-icon-right-top;
-  left: $sprk-input-has-icon-right-left;
-  margin: 0;
+  margin-top: $sprk-input-has-icon-margin-right-top;
+  margin-left: $sprk-input-has-icon-margin-right-left;
```

Comment:
this change from positional attributes to margin allows the icon to move along with the input as the `label` expands to whatever

# BEFORE:
![bottom positioning tooltip](https://user-images.githubusercontent.com/4342363/125685013-55e19231-505c-4bfb-8adc-9ed4620657f3.gif)
 size it wants to be.

# AFTER
![positioning top tooltip](https://user-images.githubusercontent.com/4342363/125685056-9a53a904-5f38-4098-9893-6552263e2016.gif)